### PR TITLE
ci: GitHub Action to detect typos with codespell

### PR DESCRIPTION
**Pull request by cclauss**
https://github.com/codespell-project/actions-codespell

### CODE REVIEW

File: .github/workflows/codespell.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# https://github.com/codespell-project/actions-codespell
+name: codespell
+on: [pull_request, push]
+jobs:
+  codespell:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: codespell-project/actions-codespell@master
```
Comment (AugustinMauroy) [inline .github/workflows/codespell.yml:9]: we should use specific version.

**Comment by AugustinMauroy** [comment]
i'm +1 for code spell

File: .github/workflows/codespell.yml (hunk 1 above)
Comment (cclauss) [reply .github/workflows/codespell.yml:9]:
```suggestion
      - uses: codespell-project/actions-codespell@v2
```

**Review by AugustinMauroy — APPROVED** [review]
LGMT !

**Pull request merged**